internal/ethapi/api.go

```diff
@@ -1335,11 +1335,14 @@ type RPCTransaction struct {
 	// L1 message transaction fields:
 	Sender     common.Address  `json:"sender,omitempty"`
 	QueueIndex *hexutil.Uint64 `json:"queueIndex,omitempty"`
+
+	// wrapped traces, currently only available for `scroll_getSkippedTransaction` API, when `MinerStoreSkippedTxTracesFlag` is set
+	Traces *types.BlockTrace `json:"traces,omitempty"`
 }
 
 // NewRPCTransaction returns a transaction that will serialize to the RPC
 // representation, with the given location metadata set (if available).
-func NewRPCTransaction(tx *types.Transaction, blockHash common.Hash, blockNumber uint64, index uint64, baseFee *big.Int, config *params.ChainConfig) *RPCTransaction {
+func NewRPCTransaction(tx *types.Transaction, traces *types.BlockTrace, blockHash common.Hash, blockNumber uint64, index uint64, baseFee *big.Int, config *params.ChainConfig) *RPCTransaction {
```

Comment:
ditto, no need to change any of the original RPC transaction logic

Reply:
fixed in https://github.com/scroll-tech/go-ethereum/pull/500/commits/8920cf9daa1f9dac2b48f9e9c485943ebd0ba890